runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/GrpcFnServer.java

```diff
@@ -76,14 +81,16 @@ public Server getServer() {
 
   @Override
   public void close() throws Exception {
-    try {
```

Comment:
Instead of overriding this method with something that handles the null. Create an anonymous subclass inside the new create() method you added which does nothing when close() is called.

Reply:
Fixed in commit https://github.com/apache/beam/pull/7168/commits/382e173a2d8aab6f045b3a213d3703b7104252b1